doc/classes/RenderingServer.xml

```diff
@@ -3026,6 +3026,14 @@
 				Sets the origin offset to be used when this reflection probe is in box project mode. Equivalent to [member ReflectionProbe.origin_offset].
 			</description>
 		</method>
+		<method name="reflection_probe_set_priority">
+			<return type="void" />
+			<param index="0" name="probe" type="RID" />
+			<param index="1" name="priority" type="int" />
+			<description>
+				Sets the priority of this probe. Equivalent to [member ReflectionProbe.priority].
```

Comment:
It's not always equivalent. A **ReflectionProbe** node may not even exist, and only be available as a RID.
I also assume that the limit of `0` to `100` applies here as well? I can't say.
```suggestion
				Sets the priority of this probe. [param priority] must be a value between [code]0[/code] and [code]100[/code]. Corresponds to [member ReflectionProbe.priority].
```

Reply:
Pretty sure I copied that from other entries, you can see the same naming all over that file. Kind of the same as your other remark about setters (which I'm pretty sure is because originally these descriptions were of the setters and simply copy/pasted/moved when proper property descriptions were added to the docs).

Agree btw on both here but we'd probably want to clean up the whole file and I think its worth doing that in a separate PR, esp as its not likely we'll be merging this PR soon.